string mittente
string destinatario
---
int64 status
string info
